src/Discord/Builders/MessageBuilder.php

```diff
@@ -632,6 +638,16 @@ public function setFlags(int $flags): self
         return $this;
     }
 
+    /**
+     * Get the current flags of the message.
+     *
+     * @return int
+     */
+    public function getFlags(): int
+    {
+        return $this->flags;
```

Comment:
the property might be not set yet

```suggestion
     * @return ?int
     */
    public function getFlags(): ?int
    {
        return $this->flags ?? null;
```

Reply:
If it's not set yet you'll also need to coalesce it to null to prevent a warning
```php
return $this->flags ?? null;
```

Reply:
I'm not actually sure this function should exist at all. Flags is a private and otherwise inaccessible property in all other similar parts of the library (member, channel, thread, message, messagebuilder). It's only been expected for the lib user to set the flag, not be checking what it's set to after the fact.

Reply:
Looking further (again) I think this is an appropriate change to the Builder classes.